tools/infrasys/Cargo.toml

```
@@ -0,0 +1,30 @@
+[package]
+name = "infrasys"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
+authors = ["Aashna Sheth <shetaash@amazon.com>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+hex = "0.4.0"
+rusoto_core = { version = "0.47", default-features = false, features = ["rustls"] }
+rusoto_cloudformation = { version = "0.47", default-features = false, features = ["rustls"] }
+tokio = { version = "1", default-features = false, features = ["macros", "rt-multi-thread"] } 
```

We've been locking tokio to an "LTS" version everywhere.

```suggestion
tokio = { version = "~1.8", default-features = false, features = ["macros", "rt-multi-thread"] } # LTS
```

---

This doesn't seem to be changed yet. We want the tokio version to be `~1.8`. And we have been putting the comment `# LTS` at the end.